src/services/resources-getter.js

```diff
@@ -1,255 +1,78 @@
 import _ from 'lodash';
-import { Schemas, logger } from 'forest-express';
-import Operators from '../utils/operators';
+import { Schemas } from 'forest-express';
 import PrimaryKeysManager from './primary-keys-manager';
-import QueryBuilder from './query-builder';
-import SearchBuilder from './search-builder';
-import LiveQueryChecker from './live-query-checker';
-import { ErrorHTTP422 } from './errors';
-import FiltersParser from './filters-parser';
+import QueryOptions from './query-options';
 import extractRequestedFields from './requested-fields-extractor';
 
 class ResourcesGetter {
-  constructor(model, options, params) {
-    this.model = model;
-    this.options = options;
-    this.params = params;
-    this.schema = Schemas.schemas[model.name];
-    this.queryBuilder = new QueryBuilder(model, options, params);
-    this.operators = Operators.getInstance(options);
-    [this.primaryKey] = _.keys(model.primaryKeys);
-    this.filterParser = new FiltersParser(this.schema, params.timezone, options);
+  constructor(model, lianaOptions, params) {
+    // The liana options argument is kept for retro-compatibility.
```

Comment:
Retrocompatibility but you are not deprecating the old way of instantiating. I think you should

Reply:
This class is instanciated from another project (forest-express).
I don't want to release a new forest-express version => that's why i kept this parameter which is now useless

Reply:
added test to be able to instanciate the new way